olca-app/src/org/openlca/app/messages_de.properties

```diff
@@ -8,6 +8,7 @@ ActivityVariable=Aktivit\u00E4tsvariable
 Actor=Kontakt
 Actors=Kontakte
 Add=Hinzuf\u00FCgen
+AddDatabase=Databank inzuf\u00FCgen
```

Comment:
It should be "hinzufügen" here. Sometimes it is maybe better to just keep the old translations.